<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CardGameClass</class>
 <widget class="QMainWindow" name="CardGameClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1763</width>
    <height>944</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CardGame</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGraphicsView" name="view1">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>270</y>
      <width>150</width>
      <height>220</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="Bt1">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>520</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Adobe Arabic</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>选择</string>
    </property>
   </widget>
   <widget class="QGraphicsView" name="view2">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>270</y>
      <width>150</width>
      <height>220</height>
     </rect>
    </property>
   </widget>
   <widget class="QGraphicsView" name="view3">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>270</y>
      <width>150</width>
      <height>220</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="Bt2">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>520</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Adobe Arabic</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>选择</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Bt3">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>520</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Adobe Arabic</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>选择</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>110</y>
      <width>461</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Adobe Arabic</family>
      <pointsize>26</pointsize>
     </font>
    </property>
    <property name="text">
     <string>CARD GAME </string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="Next_round">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>420</y>
      <width>231</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Adobe Arabic</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>下一轮</string>
    </property>
   </widget>
   <widget class="QLabel" name="Lable_mask1">
    <property name="geometry">
     <rect>
      <x>215</x>
      <y>275</y>
      <width>140</width>
      <height>210</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="Lable_mask2">
    <property name="geometry">
     <rect>
      <x>465</x>
      <y>275</y>
      <width>140</width>
      <height>210</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="Lable_mask3">
    <property name="geometry">
     <rect>
      <x>715</x>
      <y>275</y>
      <width>140</width>
      <height>210</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QPushButton" name="confirm_btn">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>570</y>
      <width>231</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Adobe Arabic</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>确认</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>340</y>
      <width>401</width>
      <height>61</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QRadioButton" name="radioButton_6">
       <property name="text">
        <string>6</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radioButton_5">
       <property name="text">
        <string>5</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radioButton_4">
       <property name="text">
        <string>4</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radioButton_3">
       <property name="text">
        <string>3</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radioButton_2">
       <property name="text">
        <string>2</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radioButton">
       <property name="text">
        <string>1</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="punishtitle_label">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>180</y>
      <width>951</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Adobe Arabic</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="text">
     <string>请选择你想要给对方减分的数量</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="punishtext_label">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>260</y>
      <width>341</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Adobe Arabic</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>对方选择给你减分：</string>
    </property>
   </widget>
   <widget class="QLabel" name="punish_label">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>260</y>
      <width>61</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>3</string>
    </property>
   </widget>
   <widget class="QLabel" name="score_label">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>100</y>
      <width>161</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Adobe Arabic</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>当前比分：</string>
    </property>
   </widget>
   <widget class="QLabel" name="A_scoretext_label">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>160</y>
      <width>81</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>A方：</string>
    </property>
   </widget>
   <widget class="QLabel" name="B_scoretext_label">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>210</y>
      <width>71</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>B方：</string>
    </property>
   </widget>
   <widget class="QLabel" name="A_score_label">
    <property name="geometry">
     <rect>
      <x>930</x>
      <y>160</y>
      <width>71</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>scoreA</string>
    </property>
   </widget>
   <widget class="QLabel" name="B_score_label">
    <property name="geometry">
     <rect>
      <x>930</x>
      <y>210</y>
      <width>71</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>scoreB</string>
    </property>
   </widget>
   <widget class="QLabel" name="givepunishtext_label">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>260</y>
      <width>341</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Adobe Arabic</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>你选择给对方减分：</string>
    </property>
   </widget>
   <widget class="QLabel" name="info_label">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>260</y>
      <width>381</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Adobe Arabic</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>休息片刻，请联系主试</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>390</y>
      <width>211</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="continue_Btn">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>480</y>
      <width>231</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Adobe Arabic</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>继续</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1763</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="CardGame.qrc"/>
 </resources>
 <connections/>
</ui>
